day_two_guide/topics/con_etcd_backup.adoc

```diff
@@ -0,0 +1,50 @@
+////
+concept about etcd backup
+
+Module included in the following assemblies:
+
+* day_two_guide/host_level_tasks.adoc
+* day_two_guide/environment_backup.adoc
+////
+
+etcd is the key value store for all object definitions, as well as the
+persistent master state. Other components watch for changes, then bring
+themselves into the desired state.
+
+{product-title} versions prior to 3.5 used etcd version 2 (v2), while 3.5 and
```

Comment:
s/used/use ? (i am assuming the prior versions still run v2).
Do we need the `, while`?  _...etcd version 2 (v2) and later versions use version 3 (v3)_